ui/src/chatbot.test.tsx

```diff
@@ -159,4 +159,72 @@ describe('XChatbot', () => {
     expect(emitMock).toHaveBeenCalledTimes(1)
     expect(emitMock).toHaveBeenCalledWith(model.name, 'scroll_up', true)
   })
+
+  it('Renders thumbs up/down buttons', () => {
+    const { container } = render(<XChatbot {...{ ...model, data, events: ['feedback'] }} />)
+    const likeButton = container.querySelector("i[data-icon-name='Like']") as HTMLLIElement
+    const dislikeButton = container.querySelector("i[data-icon-name='Dislike']") as HTMLLIElement
+    expect(likeButton).toBeInTheDocument()
+    expect(dislikeButton).toBeInTheDocument()
+  })
+
+  it('Fires a like feedback event when clicked on the thumbs up button', () => {
+    const { container } = render(<XChatbot {...{ ...model, data, events: ['feedback'] }} />)
+    const likeButton = container.querySelector("i[data-icon-name='Like']") as HTMLLIElement
+
+    fireEvent.click(likeButton)
+    expect(emitMock).toHaveBeenCalled()
+    expect(emitMock).toHaveBeenCalledTimes(1)
```

Comment:
Nit, no need for explicit check.

```suggestion
    expect(emitMock).toHaveBeenCalledTimes(1)
```